.changeset/silver-dolphins-promise.md

```diff
@@ -0,0 +1,5 @@
+---
+'@nacelle/storefront-sdk-plugin-commerce-queries': patch
+---
+
+Add `navigation` method to `@nacelle/storefront-sdk-plugin-commerce-queries`: which fetches navigation data from a Nacelle Space.
```

Comment:
The fact that it's a change to @nacelle/storefront-sdk-plugin-commerce-queries is already accounted for in the changeset, so we can simplify the language here:

```suggestion
Add `navigation` method to fetch navigation data from a Nacelle space.
```